assets/js/pages/HostRelevantPatches/HostRelevantPatchesPage.jsx

```diff
@@ -90,11 +95,11 @@ function HostRelevantPatches({ hostName, onNavigate, patches }) {
             placeholder="Search by Synopsis"
             prefix={<EOS_SEARCH size="l" />}
           />
-          <a href={file} download={`${hostName}-patches.csv`}>
+          <a href={csvURL} download={`${hostName}-patches.csv`}>
             <Button
               className="w-max"
               type="primary-white"
-              disabled={displayedPatches?.length <= 0}
+              disabled={patches.length <= 0 || !csvURL}
```

Comment:
I guess `disabled={!csvURL}` would be enough (and I hope we don't get never negative number of patches hehe)